tracer/src/Datadog.Trace/SpanTypes.cs

```diff
@@ -76,5 +76,10 @@ public static class SpanTypes
         /// The span type for serverless integrations.
         /// </summary>
         public const string Serverless = "serverless";
+
+        /// <summary>
+        /// The span type for db integrations (including couchbase)
+        /// </summary>
+        public const string Db = "db";
```

Comment:
This updates the public API, so you'll probably need to update the public API tests if you haven't already?

Reply:
Yep i've updated those (they were failing)